aws_auth.tf

```diff
@@ -78,12 +91,12 @@ resource "kubernetes_config_map" "aws_auth" {
   }
 
   data = {
-    mapRoles = yamlencode(
+    mapRoles = replace(yamlencode(
```

Comment:
Here I'm removing the quotes from the formated yaml to match the diff of kubernetes_config_map.aws_auth.